AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2583,12 +2664,18 @@ - (void)setCalculatedDisplayNodeLayout:(std::shared_ptr<ASDisplayNodeLayout>)dis
 - (CGSize)calculatedSize
 {
   ASDN::MutexLocker l(__instanceLock__);
+  if (_pendingDisplayNodeLayout != nullptr) {
```

Comment:
@maicki then why do we have this code?

+  ASDisplayNodeAssertNotNil(_pendingDisplayNodeLayout->layout, @"-[ASDisplayNode layoutThatFits:parentSize:] _pendingDisplayNodeLayout->layout should not be nil! %@", self);
 +  return _pendingDisplayNodeLayout->layout ?: [ASLayout layoutWithLayoutElement:self size:{0, 0}];